src/docchecks.jl

```diff
@@ -141,24 +143,31 @@ function footnotes(doc::Document)
             end
         end
     end
+    return
 end
 
 function footnote(fn::MarkdownAST.FootnoteLink, orphans::Dict)
     ids, bodies = get(orphans, fn.id, (0, 0))
     # Footnote references: syntax `[^1]`.
-    orphans[fn.id] = (ids + 1, bodies)
+    return orphans[fn.id] = (ids + 1, bodies)
 end
 function footnote(fn::MarkdownAST.FootnoteDefinition, orphans::Dict)
     ids, bodies = get(orphans, fn.id, (0, 0))
     # Footnote body: syntax `[^1]:`.
-    orphans[fn.id] = (ids, bodies + 1)
+    return orphans[fn.id] = (ids, bodies + 1)
 end
 footnote(other, orphans::Dict) = true
 
 # Link Checks.
 # ------------
 
-hascurl() = (try; success(`curl --version`); catch err; false; end)
+function hascurl()
+    try
+        return success(`curl --version`)
+    catch err
+        return false
+    end
+end
```

Comment:
This was also by the formatter?

Reply:
No, formatter did:
```
hascurl() = (
    try
        success(`curl --version`)
    catch err
        false
    end
)
```
(It doesn't allow single-line block constructs).